sound/soc/sof/intel/hda-loader.c

```diff
@@ -34,6 +35,8 @@
 #include "../ops.h"
 #include "hda.h"
 
+#include "../../intel/skylake/skl-sst-cldma.h"
```

Comment:
If we need any macros we should define locally.

Reply:
OK, my question is to create a new head file like hda-skl.h, or just define it in hda-loader.c now? For there are too much defines for the registers, I prefer using a new head file..

Reply:
The registers should mostly be the same as existing ones for APL, you may need to check each one being used.